serpent-moltres/full-core/createxs-full.sh

```diff
@@ -0,0 +1,75 @@
+# Step 1: Uncomment these lines to create .coe
```

Comment:
This is great. BUT, I strongly recommend a make file rather than a shell script (see the effective computation in physics book to learn why)